### ISSUE TITLE
fix: Required Switch is not Announced to Screen Readers

### ISSUE THREAD

**Issue by break-stuff** [microsoft/fast#6013]
# 🐛 Bug Report

## 💻 Repro or Code Sample

https://codesandbox.io/s/objective-vaughan-x2bbdx

## 🤔 Expected Behavior

Screen readers should announce when a switch is required.

## 😯 Current Behavior

When the `required` attribute is used on the `switch` component, it is not announced to screen readers.

## 💁 Possible Solution

Adding `aria-required="true"` to the host element when the `required` attribute is used should fix this issue.

## 🌍 Your Environment

* OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
* Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
* Version [e.g. 1.8.0]
